api_tests/files/serializers/test_file_serializer.py

```diff
@@ -1,48 +1,54 @@
-# -*- coding: utf-8 -*-
 from datetime import datetime
-from nose.tools import *  # flake8: noqa
-from pytz import utc
 
-from osf_tests import factories
-from tests.base import DbTestCase
-from tests.utils import make_drf_request_with_version
+import pytest
+from pytz import utc
 
 from api.files.serializers import FileSerializer
-
 from api_tests import utils
+from osf_tests.factories import (
+    UserFactory, 
+    NodeFactory,
+)
+from tests.utils import make_drf_request_with_version
 
+@pytest.fixture()
+def user():
+    return UserFactory()
 
-class TestFileSerializer(DbTestCase):
+@pytest.mark.django_db
+class TestFileSerializer:
 
-    def setUp(self):
-        super(TestFileSerializer, self).setUp()
-        self.user = factories.UserFactory()
-        self.node = factories.NodeFactory(creator=self.user)
-        self.file = utils.create_test_file(self.node, self.user)
+    @pytest.fixture()
+    def node(self, user):
+        return NodeFactory(creator=user)
 
-        self.date_created = self.file.versions.first().date_created
-        self.date_modified = self.file.versions.last().date_created
-        self.date_created_tz_aware = self.date_created.replace(tzinfo=utc)
-        self.date_modified_tz_aware = self.date_modified.replace(tzinfo=utc)
+    @pytest.fixture()
+    def file(self, node, user):
```

Comment:
While it is unlikely to cause problems here, `file` is actually a `type` in python like `int` or `str`. Probably just rename to file_one or something